docs/sources/http_api/serviceaccount.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Service account HTTP API"
+description = "Grafana Service account HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "serviceaccount"]
+aliases = ["/docs/grafana/latest/http_api/serviceaccount/"]
++++
+
+# Service account API
+
+This API allows you to interact programmatically with the [Service accounts]({{< relref "../manage-users/serviceaccount/_index.md" >}}).
+
+> If you are running Grafana Enterprise and have [Fine-grained access control]({{< relref "../enterprise/access-control/_index.md" >}}) enabled, for some endpoints you would need to have relevant permissions.
+> Refer to specific resources to understand what permissions are required
+
+## Delete a service account
+
+`DELETE /api/serviceaccounts/:serviceaccountId`
+
+#### Required permissions
+
+See note in the [introduction]({{< ref "#user-api" >}}) for an explanation.
```

Comment:
```suggestion
For details, see the [introduction]({{< ref "#user-api" >}}).
```

Reply:
@achatterjee-grafana, do all references need to use `relref`? @GrafanaWriter and @chri2547, this question is something that we ought to include in our Markdown conventions at https://github.com/grafana/grafana/blob/main/contribute/style-guides/documentation-markdown-guide.md.

Reply:
For internal content within the same repo, yes.